src/models/Collaborator.ts

```diff
@@ -1,6 +1,8 @@
 import { MyContext } from '../context';
 import { validateEmail } from '../utils/helpers';
 import { MySqlModel } from './MySqlModel';
+import { Template } from './Template';
+import { User } from './User';
```

Comment:
L9: I really liked the detailed comments you included in MySQLModel.ts, like this one:
 ```
 // Run a query to check for the existence of a record in the database. Typically used to verify that
  // a foreign key relationship exists. (e.g. TemplateCollaborator runs a check to make sure the
  // templateId exists before creating a new record)
  //    - apolloContext:   The Apollo server context
  //    - tableName:       The name of the tabe to query
  //    - id:              The id of the record we are looking for
  //    - reference:       A reference to contextualize log messages e.g. `users resolver`
```
I'm wondering if you can add some JSDocs to these models to help keep track of what the variables are.